app-text/stardict/stardict-3.0.6-r4.ebuild

```
@@ -161,8 +162,9 @@ src_install() {
 		dodoc tools/{AUTHORS,ChangeLog,README}
 
 		if use examples; then
-			insinto /usr/share/doc/${PF}/tools
-			doins tools/src/{dictbuilder.{example,readme},example.ifo,example_treedict.tar.bz2}
+			docinto tools
+			dodoc tools/src/{dictbuilder.{example,readme},example.ifo,example_treedict.tar.bz2}
+			docompress -x /usr/share/doc/${PF}/tools/{dictbuilder.{example,readme},example.ifo,example_treedict.tar.bz2}
```

Can't you just add the whole directory?
```suggestion
			docompress -x /usr/share/doc/${PF}/tools
```

---

Yes, this would work. The only downside, why I initially did that, as there are other text files before installed, which I didn't wanted to exclude from compressing. But for better readability, I guess, it's okay in this case. Done.